client/src/sass/application.scss

```diff
@@ -97,7 +97,6 @@ label {
   color: #000;
   font-size: 16px;
   display: inline-block;
-  margin-right: 55px;
```

Comment:
Did you check it does not break things in the admin pages?

Reply:
Id kinda does now that you mention it :frowning: